### ISSUE TITLE
How to pass in custom TXT?

### ISSUE THREAD

**Issue by pastawoua** [ardean/spread-the-word#2]
When I start the broadcast, I can see a default TXT record `0=0`, is there a way to pass in custom records please?

**Comment by ardean** [comment]
The demo has been updated.

The Api looks like this:
```ts
await stw.spread({
  name: "remote receiver",
  port: 4444,
  type: "jsremote",
  txt: {
    myProp: "Hello"
  }
});
```

And the remote service:
```ts
stw
  .on("up", (remoteService: RemoteService) => {
    if (remoteService.txt) console.log("TXT found:", remoteService.txt.myProp);
  })
  .on("down", (remoteService: RemoteService) => {
    if (remoteService.txt) console.log("TXT found:", remoteService.txt.myProp);
  });
```
Please let me know if something is not working.

**Comment by pastawoua** [comment]
@ardean awesome, thanks! Looks like it fixed #1 too.
